xml/networking-neutron_troubleshooting.xml

```diff
@@ -8,25 +8,23 @@
  <para>
   Troubleshooting scenarios with resolutions for the Networking service.
  </para>
- <section xml:id="network-fail-troublshoot">
-  <title>Troubleshooting Network failures</title>
-  <para>
-   <emphasis role="bold">CVR HA - Split-brain result of failover of L3 agent
-   when master comes back up</emphasis> This situation is specific to when L3
-   HA is configured and a network failure occurs to the node hosting the
+
+   <section>
+     <title>CVR HA - Split-brain result of failover of L3 agent when master comes back up</title>
+     <para>This situation is specific to when L3 HA is configured and a network failure occurs to the node hosting the
    currently active l3 agent. L3 HA is intended to provide HA in situations
    where the l3-agent crashes or the node hosting an l3-agent crashes/restarts.
    In the case of a physical networking issue which isolates the active l3
    agent, the stand-by l3-agent takes over but when the physical networking
    issue is resolved, traffic to the VMs is disrupted due to a "split-brain"
    situation in which traffic is split over the two L3 agents. The solution is
-   to restart the L3-agent that was originally the master.
-  </para>
-  <para>
-   <emphasis role="bold">OVSvApp loses connectivity with vCenter</emphasis> If
-   the OVSvApp loses connectivity with the vCenter cluster, you will receive
-   the following errors:
-  </para>
+   to restart the L3-agent that was originally the master.</para>
+ </section>
+
+   <section>
+     <title>OVSvApp loses connectivity with vCenter</title>
```

Comment:
titles might need different capitalization (StyleGuide section 4.2). Not sure if that applies in the present situation.